docs/backend-api/resources/tracking/delivery.md

```diff
@@ -1,9 +1,11 @@
 ---
-title: /delivery
-description: /delivery
+title: Delivery info
+description: Delivery info
 ---
 
-## read(…)
+## Get info for specified delivery
```

Comment:
The documentation structure for the REST API assumes that we list all possible calls to a specific resource. Therefore, the call itself must be a second level heading. "Get info for specified delivery" - it's just description, not title.

Reply:
I wanted to know this moment in more detail:
1. How can we change the left menu name for the call? I thought it changes with title and description at the beginning of the document. 
https://i.imgur.com/9U3Kkwv.png
2. How can we change the right menu name for the method? I thought it changes if I use different leveled headers. Like this - ##read(...) -> ## Get info for specified delivery and using ###read to make it lower in the right menu
https://i.imgur.com/OlqPtwu.png

Reply:
1. The attribute `title` in meta (between `---` in top) is responsible for this.
2. Yep, the right menu is formed by the headings on the page. For consistency and uniformity, the methods for working with the described resource (entity) of the API should be placed in the headers of the second level. The description of these methods should go in plain text below. We can think about removing the `(...)` in the designation of methods. But for now, I suggest to keep it.